app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,20 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(
+            self,
+            nickname: str,
+            favourite_dish: str,
+            hummer_level: int
+    ) -> None:
+        super().__init__(nickname, favourite_dish)
+        self._hummer_level = hummer_level
```

Comment:
The attribute `_hummer_level` should likely be `_hammer_level` to correct the typo and maintain consistency with the intended meaning.